manifests/p/Prezi/PreziClassic/6.26.0/Prezi.PreziClassic.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate
+
+PackageIdentifier: Prezi.PreziClassic
+PackageVersion: 6.26.0
+# MinimumOSVersion:
+Platform:
+- Windows.Desktop
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://prezi.com/download
```

Let's change this to the direct link for the version. The current InstallerUrl (https://prezi.com/download) only redirects to the latest version, so when a new version comes out, this manifest will no longer work.

```suggestion
  InstallerUrl: https://desktopassets.prezi.com/prezi-desktop/win/Releases/PreziClassic_6.26.0.exe
```